Sources/EpoxyLayoutGroups/Groups/VGroup.swift

```diff
@@ -89,20 +90,24 @@ public final class VGroup: UILayoutGuide, Constrainable, InternalGroup {
   public struct Style: Hashable {
     public init(
       alignment: VGroup.ItemAlignment = .fill,
-      spacing: CGFloat = 0)
+      spacing: CGFloat = 0,
+      springAnimationParameters: SpringAnimationParameters = SpringAnimationParameters())
     {
       self.alignment = alignment
       self.spacing = spacing
+      self.springAnimationParameters = springAnimationParameters
     }
 
     let alignment: VGroup.ItemAlignment
     let spacing: CGFloat
+    let springAnimationParameters: SpringAnimationParameters
   }
 
   /// Alignment set at the group level to apply to all constrainables
   /// Individual alignments on a constrainable take precedence over the group's alignment.
   /// The default value is `.fill`
   public let alignment: ItemAlignment
+  public let springAnimationParameters: SpringAnimationParameters
```

Comment:
can we include the same comment here that we have in `HGroup`?

Reply:
Sure thing. Done!